spec/system/storage_location_system_spec.rb

```diff
@@ -92,6 +92,23 @@
       end
     end
 
+    it "displays the correct grand totals across all storage locations" do
```

Comment:
This should be a request spec, not a system spec since there's no interaction. Request specs are a lot faster and more lightweight.

Reply:
Is this still needed now that we have the request spec?